keystrokes/src/components/Home/Home.js

```diff
@@ -1,16 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Game from '../Game/Game';
 import Nav from '../NavBar/NavBar';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import { ref } from '../../config/constants';
+import CharacterList from '../Character/CharacterList';
+
+function Home(props) {
+	const userId = props.user.email.replace(/[.$#[\]]/g, '');
+	const userPath = `/Users/${userId}`;
+	const [user, setUser] = useState({
+		characters: {},
+		currentCharacter: null,
+	});
+
+	useEffect(() => {
+		const collectUserInfo = async () => {
```

Comment:
I'm really curious why you are reluctant to use named functions 
`async function collectUserInfo () {`

Reply:
this use effect is 👌

Reply:
i think when i first started in javascript i just thought `const myCoolFuncion = () => {` was just a cool way to write functions, so i kinda built it as a habit.